### ISSUE TITLE
Architektura CNN klasyfikatora (7 klas, Wigner/Husimi)

### ISSUE THREAD

**Issue by barkol** [e-mcQuantum-AI/IDUB_2026#46]
4 bloki Conv+BN+ReLU+Pool + 2 FC + Softmax. Wariant: ResNet-18 fine-tuned. Wejście (1,64,64). PLAN_ROZWOJU.md §3.1

---
_Originally tracked as #40 in barkol/IDUB_2026_
